.gitignore

```diff
@@ -10,12 +10,18 @@
 /build
 
 # misc
-**/.DS_Store
-**/.env.local
+.DS_Store
+.env.local
 .env.development.local
 .env.test.local
 .env.production.local
```

Comment:
i dont get it, tu vas passer les .env comment ?

Reply:
Dans la conf du CI et du server. 
En local, on foutra un `.env.local.sample` limite. 
Mais ce gitignore c'est un gitignore standard que j'ai recup

Reply:
Parce que la je commit un `.env` à cote du `docker-compose.yml` qui fait office de setup des variables d'env